index.md

```diff
@@ -1,125 +1,93 @@
 2. ML experiments
 3. Example use cases for ML models in the real world
 
-# Featured Models
-<div class="github-card" data-github="onnx/models" data-width="400" data-height="" data-theme="default"></div>
-<script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>
+<br/>
+<br/>
+* * *
+# Featured
 
-# Featured Examples
-We can prettify this, or just point to https://github.com/search?q=machinelearningsamples (none of these repos are properly tagged with topics...)
 
-# Top GitHub Content (Discover Experience)
-## Topic Search: 
-<input type='text' id='keyword' name='keyword' value='mnist'>
-<input type="button" id="btnSearch" value="Search"/>
-<div class="github-widget" data-type="search"></div>
+## Models
 
-## Browse Top Repos
-<div id="browse_content">
-    <div class="browse_result" style="display:inline-block; width:200; vertical-align: top">
-        <div class="github-widget" data-type="browse" data-browseTopic="onnx"></div>
-    </div>
-    <div class="browse_result" style="display:inline-block; width:200; vertical-align: top">
-        <div class="github-widget" data-type="browse" data-browseTopic="tensorflow"></div>
+<div class="menu-wrapper" id="model-wrapper">
+    <ul class="menu models" id="models-menu">
+    </ul>
+    <div class="paddles">
+        <button class="left-paddle paddle hidden" id="model-left"><</button>
+        <button class="right-paddle paddle" id="model-right">></button>
     </div>
 </div>
 
-<script src="./dist/bundle.js"></script>
-
-# Header 1
-
-This is a normal paragraph following a header. GitHub is a code hosting platform for version control and collaboration. It lets you and others work together on projects from anywhere.
+* * *
 
-## Header 2
 
-Code example below. Try this new functionality!
+<br/>
+## Examples
+<div class="menu-wrapper" id="example-wrapper">
+    <ul class="menu examples" id="examples-menu">
+    </ul>
+    <div class="paddles">
+        <button class="left-paddle paddle hidden" id="example-left"><</button>
+        <button class="right-paddle paddle" id="example-right">></button>
+    </div>
+</div>
 
-```js
-// Javascript code with syntax highlighting.
-var fun = function lang(l) {
-  dateformat.i18n = require('./lang/' + l)
-  return true;
-}
-```
+* * *
+<br/>
 
-### Lists of Content
 
-*   This is an unordered list following a header.
-*   This is an unordered list following a header.
-*   This is an unordered list following a header.
+# Discover
+## Search Topics: 
 
-1.  This is an ordered list following a header.
-2.  This is an ordered list following a header.
-3.  This is an ordered list following a header.
+<div class="content-container">
+    <input type="button" id="btnSearch" value="Search" class="button btnSearch  searchbtn noselect"/>
+    <div  class="searchbar"><input type='text' id='keyword' name='keyword' value='mnist' class="searchbar" max-width="100%"></div>
+</div>
 
-###### Header 6
+<div class="github-widget" data-type="search"></div>
```

Comment:
so the search is still against all public github repos, not limited by Azure org, right?

Reply:
This is correct, the search is against all git repos

Reply:
I have a separate feature for limiting the search